frontend/src/app/modules/boards/board/inline-add/board-inline-add-autocompleter.component.ts

```diff
@@ -51,6 +51,7 @@ import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
 import {WorkPackageCardDragAndDropService} from "core-components/wp-card-view/services/wp-card-drag-and-drop.service";
 import {WorkPackageNotificationService} from "core-app/modules/work_packages/notifications/work-package-notification.service";
 import {UrlParamsHelperService} from "core-components/wp-query/url-params-helper";
+import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/cfe89dd706c72154281f341454501767d598e770/frontend/src/app/modules/boards/board/inline-add/board-inline-add-autocompleter.component.ts#L55"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>